src/transformers/models/vilt/modeling_vilt.py

```diff
@@ -969,7 +972,8 @@ def forward(
 
         >>> selected_token = ""
         >>> encoded = processor.tokenizer(inferred_token)
-        >>> processor.decode(encoded.input_ids[0], skip_special_tokens=True)
+        >>> output = processor.decode(encoded.input_ids[0], skip_special_tokens=True)
+        >>> print(output)
```

Comment:
Think it is better to `print(output)` here.

Reply:
We can add an expected output here and add ViLT to the doc tests

Reply:
There is already an expected output. That's why I think it is better to print the output.
Otherwise, the code example won't show any output, and therefore the expected output seems (logically) strange.
(I am not sure if this is a valid argument though)

Reply:
Yes, if we print something it should be shown below :-)